.github/workflows/build_publish.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           if [[ $BRANCH == "main" ]]
           then
             BUILD_NUMBER=$GITHUB_RUN_NUMBER
-            VERSION="1.0.${BUILD_NUMBER}"
+            VERSION="2.0.${BUILD_NUMBER}"
```

Comment:
What's that bump for?

Reply:
This is a breaking change, thus we need to maintain some appearances at the very least.